share/spice/cryptocurrency/cryptocurrencylist.txt

```diff
@@ -219,7 +219,7 @@ enrg,energycoin,energy coin,
 eqx,equinoxcoin,equinox coin,
 erm,ermiscoin,ermis coin,
 etc,eternalcoin,eternal coin,
-eth,etherium,etherium coin,
+eth,ether,etherium,etherium coin,ethereum,
```

Comment:
@tagawa True. We should probably update `etherium coin` to `ethereum coin` though (or add another trigger).